sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -1,70 +1,52 @@
 import pytest
+import datetime
 
 from sunpy.time.timerange import TimeRange
-from sunpy.net.vso.attrs import Time, Instrument
+from sunpy.net.vso.attrs import Time, Instrument, Physobs
 from sunpy.net.dataretriever.client import QueryResponse
 import sunpy.net.dataretriever.sources.goes as goes
 from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
 from sunpy.net import Fido
 from sunpy.net import attrs as a
 
-LCClient = goes.GOESClient()
-
-@pytest.mark.parametrize("timerange,url_start,url_end",
-[(TimeRange('1995/06/03', '1995/06/05'),
-'http://umbra.nascom.nasa.gov/goes/fits/1995/go07950603.fits',
-'http://umbra.nascom.nasa.gov/goes/fits/1995/go07950605.fits'),
-(TimeRange('2008/06/02', '2008/06/04'),
-'http://umbra.nascom.nasa.gov/goes/fits/2008/go1020080602.fits',
-'http://umbra.nascom.nasa.gov/goes/fits/2008/go1020080604.fits')
-])
-def test_get_url_for_time_range(timerange, url_start, url_end):
-    urls = LCClient._get_url_for_timerange(timerange)
-    assert isinstance(urls, list)
-    assert urls[0] == url_start
-    assert urls[-1] == url_end
```

Comment:
This was for when GOES accepted no extra arguments. But now it does accept extra ones, namely the Physical observation (and also time range, since we might also download real-time data). These tests wouldn't remain valid and hence have been removed and new ones have been added which test and reflect GOES's new downloading logic.